operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ClusterStatus.java

```diff
@@ -38,6 +38,11 @@
   @Range(minimum = 0)
   private Integer maximumReplicas;
 
+  /** The requested number of cluster members. Required. */
+  @Description("The requested number of cluster members. Required.")
```

Comment:
Let's discuss the use of "Required" here.  I think it's okay because we are going to version 3.0.0, so there can be non-backward compatible changes. However, just having "Required" in the description doesn't make the field required.  If we want the field to be required in the schema, then tag the field with @NonNull.

Reply:
I don't think it should be required - seems too prescriptive. I suspect what Lenny really meant was that the default is 0, which surprises some folks.  How about something like "This value must be set above 0 (the default) to ensure cluster members start."?   Not sure if that's good wording, but maybe something like that?

Reply:
Updated replicasGoal to:

  /** The requested number of cluster members from the domain spec. */
  @Description("The requested number of cluster members from the domain spec. "
      + "Cluster members will be started by the operator if this value is larger than zero.")
  @Range(minimum = 0)
  private Integer replicasGoal;